### PR TITLE
Refactoring Assessment Report struct

### PR DESCRIPTION
**Pull request by sanyamsinghal**
Refactor: MigrationComplexity field to be present AssessmentReport and SchemaReport at top level
- previously it was nested inside the structs
- also updated the expected test outcomes, callhome payload struct

Refactor: move SchemaSummaryDBObjectsDesc from AssessmentReport struct to SchemaSummary struct

Contributes to https://yugabyte.atlassian.net/browse/DB-13348

### CODE REVIEW

**Comment by sanyamsinghal** [comment]
https://jenkins.dev.yugabyte.com/job/users/job/yb-voyager-testing/job/yb-voyager-testing-pipeline/3864/

File: yb-voyager/cmd/assessMigrationCommand.go [hunk 1]
```diff
@@ -801,10 +801,11 @@ func generateAssessmentReport() (err error) {
 
 func getAssessmentReportContentFromAnalyzeSchema() error {
 	schemaAnalysisReport := analyzeSchemaInternal(&source)
+	assessmentReport.MigrationComplexity = schemaAnalysisReport.MigrationComplexity
 	assessmentReport.SchemaSummary = schemaAnalysisReport.SchemaSummary
-	assessmentReport.SchemaSummaryDBObjectsDesc = "Objects that will be created on the target YugabyteDB."
+	assessmentReport.SchemaSummary.Description = "Objects that will be created on the target YugabyteDB."
```
Comment (priyanshi-yb) [inline yb-voyager/cmd/assessMigrationCommand.go:806]: This can now also go in analyze-schema now right?

**Review by priyanshi-yb — APPROVED** [review]
LGTM with a comment

File: yb-voyager/cmd/assessMigrationCommand.go (hunk 1 above)
Comment (sanyamsinghal) [reply yb-voyager/cmd/assessMigrationCommand.go:806]: yeah but i thought to follow - keep the description of a struct inside that struct itself.

Comment (priyanshi-yb) [reply yb-voyager/cmd/assessMigrationCommand.go:806]: @sanyamsinghal, No, I meant we are not populating the `SchemaSummary.Description` in the analyze schema report. So should we that as well ?

**Pull request merged**